cmplibrary/src/main/java/com/sourcepoint/cmplibrary/campaign/CampaignManager.kt

```diff
@@ -413,22 +421,10 @@ private class CampaignManagerImpl(
             val hasNonEligibleLocalDataVersion =
                 dataStorage.localDataVersion != DataStorage.LOCAL_DATA_VERSION_HARDCODED_VALUE
 
-            val res = (isGdprOrCcpaUuidPresent && isLocalStateEmpty) ||
+            return (isGdprOrCcpaUuidPresent && isLocalStateEmpty) ||
```

Comment:
since this part doesn’t impact this pr, can you please revert it back?

Reply:
Without this log, I'm not able to check if the other use cases are working correctly